dtm0/svc_internal.h

```diff
@@ -0,0 +1,111 @@
+/* -*- C -*- */
+/*
+ * Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For any questions about this software or licensing,
+ * please email opensource@seagate.com or cortx-questions@seagate.com.
+ *
+ */
+
+
+#pragma once
+
+#ifndef __MOTR_DTM0_SVC_INTERNAL_H__
+#define __MOTR_DTM0_SVC_INTERNAL_H__
+
+/* import */
+#include "fop/fom_long_lock.h"       /* m0_long_lock */
+#include "rpc/link.h"                /* m0_rpc_link */
+struct m0_dtm0_service;
+
+/* export */
+struct dtm0_process;
+
+
+/* Settings for RPC connections with DTM0 services. */
+enum {
+	DTM0_MAX_RPCS_IN_FLIGHT = 10,
+	DTM0_DISCONNECT_TIMEOUT_SECS = 1,
+};
+
+
+/**
+ * System process which dtm0 subscribes for state updates
```

Comment:
Whose state updates?

Reply:
Rewritten in https://github.com/Seagate/cortx-motr/pull/802/commits/dc9e306a71149d62a4758dd1b66a24db7af684ad#diff-99f2bf96496114a3cb8bada974e942746eeea8675036ea716403c98a599ce087